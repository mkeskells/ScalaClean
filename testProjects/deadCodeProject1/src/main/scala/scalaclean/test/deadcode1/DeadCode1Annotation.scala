package scalaclean.test.rules.deadcode.deadannotation

object App1 extends App {}
case class UnusedClass(value: String)

@deprecated
case class UnusedClassWithAnnotation(value: String)
/** some docs */
object UnusedObjectDoc