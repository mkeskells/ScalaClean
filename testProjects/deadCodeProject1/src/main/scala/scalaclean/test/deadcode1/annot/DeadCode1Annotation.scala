package scalaclean.test.deadcode1.annot
import java.lang.annotation.Inherited

object AnnotApp1 extends App {}

case class UnusedClass(value: String)

@deprecated
case class UnusedClassWithAnnotation(value: String)

/** some docs */
object UnusedObjectDoc

@Inherited
@deprecated
case class UnusedClassWithAnnotation2(value: String)

/** some docs */
@Inherited
@deprecated
object UnusedObjectDoc2

//some comments
@Inherited
//some comments
@deprecated
//some comments
case class UnusedClassWithAnnotation3(value: String)

//some comments
/** some docs */
@Inherited
//some comments
@deprecated
//some comments
object UnusedObjectDoc3
