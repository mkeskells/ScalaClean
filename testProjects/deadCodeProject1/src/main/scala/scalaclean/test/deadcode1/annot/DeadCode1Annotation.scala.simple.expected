package scalaclean.test.deadcode1.annot
import java.lang.annotation.Inherited

object AnnotApp1 extends App {}
